src/components/Home.js

```diff
@@ -13,8 +13,42 @@ import '../css/Home.css';
 import Button from 'react-bootstrap/Button';
 import Carousel from 'react-bootstrap/Carousel';
 import TextLoop from "react-text-loop";
+// import { useLocation } from "@reach/router";
+import AffiliateBanner from './AffiliateBanner';
 
-function Home() {
+function Home(props) {
+
+  const { collab } = props;
+
+  //Method:URLSearchParams
+  //Tested with url: 'soulsmile.club/tab/?affiliate=girlfriend'
+
+
+  //Attempt 1 - issue: Error: useLocation hook was used but a LocationContext.Provider was not found in the parent tree. Make sure this is used in a component that is a child of Router
+  // function useQuery() {
```

Comment:
I can look into this with you later to see why it didn't work! Thanks for trying though :) In general, it might be better to make a separate branch for attempts that didn't work so we can discuss them together (rather than leaving them commented out in the master branch)!

Reply:
Makes sense :) will definitely do that moving forward